src/uct/ib/ud/accel/ud_mlx5.c

```diff
@@ -644,7 +647,9 @@ static UCS_CLASS_INIT_FUNC(uct_ud_mlx5_iface_t,
     UCS_CLASS_CALL_SUPER_INIT(uct_ud_iface_t, &uct_ud_mlx5_iface_ops,
                               md, worker, params, 0, config);
 
-    uct_ib_iface_set_max_iov(&self->super.super, 1);
+    uct_ib_iface_set_max_iov(&self->super.super,
+                             (UCT_UD_MLX5_WQE_SIZE - sizeof(struct mlx5_wqe_ctrl_seg)) /
```

Comment:
UCT_UD_MLX5_WQE_SIZE is only the size of ctrl+address, it is not the maximal possible size
the maximal size is (UCT_IB_MLX5_MAX_BB * MLX5_SEND_WQE_BB). and need to do something like:
((UCT_IB_MLX5_MAX_BB * MLX5_SEND_WQE_BB) - sizeof(ctrl)-sizeof(dgram)-sizeof(inl) -max_hdr) / sizeof(dseg)
(i noticed that max_inline was also calculated incorrectly in ud_mlx5)